src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts

```diff
@@ -102,4 +116,15 @@ footerAlign() {
     };
   }
 
+  getTenantConfig() {
+    const slug = this.userService.slug ? this.userService.slug : this.defaultTenant;
+    this.tenantService.getSlugDefaultTenantInfo(slug, this.defaultTenant).pipe(takeUntil(this.unsubscribe$)).subscribe(
```

Comment:
Change getSlugDefaultTenantInfo => getTenantConfig

Reply:
-A function with the name is already defined in tenant.service.ts
- refer https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/3878/files#diff-1ea5535934253521c15e7f1270161753R81